homework.js

```diff
@@ -0,0 +1,126 @@
+
+const summation = function(n)
+
+{
+let sum = 0;
+
+for (let i = 1; i <= n; i=i+1) {
+
+    sum += i;//1  //3 //6 //
+}
+    return sum;
+
+}
+//summation(5);15
+
+//2.....................................................
+const summationEven = function (n)
+{
+    let summation = 0;
+
+    for (let i = 0; i < n ; i=i+1) {
+
+        if ( i %2 ===0){
+
+         summation += i;
+
+} return summation;
+}
+// summationEven(5)
+//6
+//3.......................................
+const avg = function (arry){
+    let summation = 0;
+    for (let i = 0; i < arry.length; i=i+1) {
+
+        summation = summation + arry[i];
+       summation= summation/arry.length;
+} return summation;
+}
+//[8, 2, 2, 4] = 4
+
+
+
+//4....................................................
+
+function reversWord(w) {
+
+var newString = "";
+for (let i = w.length - 1; i >= 0; i--) {
+newString += w[i];
```

Comment:
w is a string not an array, if you want to access it as an array use the method "string.charAt( i )" not string[ i ]